Project/Assets/Prairie/Framework/Script/Twine/TwineNode.cs

```diff
@@ -31,24 +64,44 @@ public void OnGUI()
 		if (this.enabled && !this.isMinimized) {
 
 			float frameWidth = Screen.width / 3;
-			Rect frame = new Rect (10, 10, frameWidth, Screen.height);
+			Rect frame = new Rect (10, 10, frameWidth, Screen.height / 2);
```

Comment:
It'd be great to introduce max sizes here - on big screens, or projectors (which we'll be demoing on), half of the screen is a huge amount more space than really needed.